### PR TITLE
Add a thermal scanner mission

### PR DESCRIPTION
**Pull request by TheGiraffe3**
As requested in https://github.com/endless-sky/Endless-Sky-Delta/discussions/15#discussioncomment-10808682.

### CODE REVIEW

File: data/human/free worlds side plots.txt [hunk 1]
```diff
@@ -426,6 +426,86 @@ ship "Fury" "Doombat"
...
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			`	You give your name to the receptionist, and she directs you to the hangar where the weapons group does their design and prototyping. Barmy Edward shakes your hand warmly and says, "We've got a new scanner that theoretically will scan ships and give information on the heat levels inside them. Are you interested in helping us test this?"`
+			choice
+				`	"Sure!"`
+				`	"No thanks."`
+					decline
+			`	"We need you to fly to <waypoints> and disable the Doombat, which is still armed with flamethrowers. Another <npc model>, the <npc>, will fly by you with the new scanners installed and take readings on the battle. Then come back here and report."`
```
Comment (TheGiraffe3) [inline data/human/free worlds side plots.txt:464]:
```suggestion
			`	"We need you to fly to <waypoints> and disable the Doombat, which is still armed with flamethrowers. Another <npc model>, the <npc>, will fly by you with the new scanners installed and take readings on the battle. Then come back here and report.`
```

**Comment by Zitchas** [comment]
First and second glance look good. Once I get the chance to play through it, I'll merge this. Should be <24h.

File: data/human/free worlds side plots.txt [hunk 2]
```diff
@@ -426,6 +426,86 @@ ship "Fury" "Doombat"
...
+			`A message arrives for you soon after landing the <ship> on <origin>. It's from Barmy Edward, the weapons engineer at Kraz Cybernetics. He writes: "Hello again, Captain! I've been working on something new, and for some reason no one else wants to help out with the testing. If you're interested in helping drop by <destination> and come visit our factory."`
+				accept
+
+
+mission "FW Thermal Scanner 2"
+	landing
+	name "Scanner Testing"
+	description "A test ship with flamethrowers is in the <waypoints> system. Fight and disable it, keeping <npc> safe, and then go back to <destination> to give Barmy Edward the scanner logs."
+	source "Rust"
+	to offer
```
Comment (Zitchas) [inline data/human/free worlds side plots.txt]:
```suggestion
	source "Rust"
	waypoint "Ildaria"
	to offer
```

Without this, the mission description text just says "A test ship with flamethrowers is in the <waypoints> system." without specifying where that is. Nor does it provide any kind of indicator without having this specified.

**Review by Zitchas — CHANGES_REQUESTED** [review]
One change that needs to be done so that the mission text lists the waypoint properly and the player knows where to go, other than that it looks good.

I feel like there's something I'm missing, some detail in the back of my head, but I can't put my finger on it. With the addition of the waypoint line the missions all work properly.

**Review by Zitchas — APPROVED** [review]
Looks good now. Thanks!

**Comment by TheGiraffe3** [comment]
You merged that. Might it be good to revert and squash?

**Comment by Zitchas** [comment]
Thanks for spotting that. Repository has been hard reset to prior to doing so. Could you re-PR please?

**Pull request merged**